### PR TITLE
SDIT-1820 Increase prod threads and refactor

### PR DESCRIPTION
**Pull request by andymarke**
Minor refactor for a few things that I struggled with, e.g. listener name

### CODE REVIEW

File: helm_deploy/hmpps-prisoner-events/values.yaml [hunk 1]
```diff
@@ -36,6 +36,9 @@ generic-service:
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
     JMS_CONNECTION_ORACLE_TRACELEVEL: 0
+    JMS_CONNECTION_CONCURRENTCONSUMERS: 10
+    JMS_CONNECTION_MAXCONCURRENTCONSUMERS: 20
+    SPRING_DATASOURCE_HIKARI_MAXIMUM_POOL_SIZE: 21
```
Comment (andymarke) [inline helm_deploy/hmpps-prisoner-events/values.yaml:42]: all environments up to 10 threads

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/config/JMSConfiguration.kt [hunk 2]
```diff
@@ -11,7 +11,7 @@ import org.springframework.jms.core.JmsTemplate
 import org.springframework.jms.listener.DefaultMessageListenerContainer
 import org.springframework.jms.support.destination.JmsDestinationAccessor
 import org.springframework.util.ErrorHandler
-import uk.gov.justice.digital.hmpps.prisonerevents.service.JMSReceiver
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/config/JMSConfiguration.kt:14]: rename to be consistent with other projects

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/repository/SqlRepository.kt [hunk 3]
```diff
@@ -9,23 +9,43 @@ import java.sql.Types
 import java.time.LocalDate
 import java.time.LocalTime
 
-const val LIMIT = 500
-
 @Repository
 class SqlRepository(private val jdbcTemplate: NamedParameterJdbcTemplate) {
 
   fun getNomsIdFromOffender(offenderId: Long): Collection<String> = jdbcTemplate.query(
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/repository/SqlRepository.kt:15]: inline all these for readability

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/service/XtagEventsListener.kt [hunk 4]
```diff
@@ -7,28 +7,29 @@ import jakarta.jms.MessageListener
...
   override fun onMessage(message: Message) {
-    xtagEventsService.addAdditionalEventData(
-      offenderEventsTransformer.offenderEventOf(
-        message as AQjmsMapMessage,
-      ),
-    )?.also {
-      eventsEmitter.sendEvent(it)
-      log.info(it.toString())
-    }
+    val aqMessage = message as AQjmsMapMessage
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/service/XtagEventsListener.kt:22]: swapped to functional chaining for readability

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/service/transformers/OffenderEventsTransformer.kt [hunk 5]
```diff
@@ -73,7 +73,7 @@ class OffenderEventsTransformer {
       log.warn("Bad xtag: {}", xtag)
       return null
     }
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/service/transformers/OffenderEventsTransformer.kt:75]: currently prints "Xtag Xtag"

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**